### DISCUSSION TITLE
How to remove the coercion warning resulted in R to HTML?

### DISCUSSION THREAD

**Discussion by DGUM27** [teamalgoritma/community#286] [Q&A] [answered]
Dear Bapak/Ibu, 

Saya mengalami issue ## Warning: Coercing yang muncul di HTML sebagaimana gambar terlampir. Mohon bantuannya untuk me-remove-nya. 

Terima kasih.

Attachment:
![a](https://user-images.githubusercontent.com/107378736/187595927-2f2c68c0-5ba8-40a8-979f-b456d2f41f3f.png)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @DGUM27 

Pesan warning biasanya memang cukup panjang dan terkadang menganggu untuk dilihat pada hasil knit Rmarkdown kita. Untuk menghilangkan pesan warning Anda dapat mengganti kode berikut yang biasanya {r} menjadi {r message=FALSE, warning=FALSE} di atas chunk yang menyebabkan kode warning, dalam hal chunk pembacaan data FSR Anda.

Contohnya seperti ini
![image](https://user-images.githubusercontent.com/56690035/187817431-9421648c-3e4a-4ac2-bc02-15fba28bb662.png)

Sekian, semoga jawaban ini membantu Anda
